packages/dashboard-core-plugins/src/panels/ChartPanel.tsx

```diff
@@ -786,20 +785,22 @@ export class ChartPanel extends Component<ChartPanelProps, ChartPanelState> {
    * Set chart filters based on the filter map
    * @param filterMapParam Filter map
    */
-  setFilterMap(
-    filterMapParam: Map<string, { columnType: string; value: string }>
-  ): void {
+  setFilterMap(filterMapParam: LinkFilterMap): void {
     log.debug('setFilterMap', filterMapParam);
     this.setState(state => {
       const { columnMap, filterMap } = state;
-      let updatedFilterMap: null | Map<string, string> = null;
+      let updatedFilterMap: null | FilterMap = null;
       const filterValueMap = new Map(state.filterValueMap);
-
-      filterMapParam.forEach(({ columnType, value }, columnName) => {
+      filterMapParam.forEach(({ columnType, filterList }, columnName) => {
         const column = columnMap.get(columnName);
         if (column == null || column.type !== columnType) {
           return;
         }
+        if (filterList.length < 1) {
+          log.error('Invalid filterMap, filterList is empty', filterMapParam);
```

Comment:
This is just going to return for this specific param (`{ columnType, filterList }`) and the `.forEach` is going to continue. You should either include the `columnType` in the log line to indicate exactly what is failing, or throw here to get out of the `.forEach`, or use a different for loop type so you can just return.

Reply:
We don't throw or break out of the loop when the column is null in the condition above, I thought it was ok to just skip the `filterMap`.
Should I throw in the condition above as well?

Reply:
I'm fine with skipping it, but the error logging should be consistent. In the above block, we're not logging anything; in this block, we're logging the entire `filterMapParam`, and not the `columnType` or `columnName` (which is unique to this iteration of the loop).
I'd just do a log debug:
```
log.debug('Ignoring empty filterList for column', columnName);
```